python/pyspark/pandas/tests/indexes/test_datetime.py

```diff
@@ -101,22 +102,8 @@ def test_properties(self):
                 self.assert_eq(psidx.day_of_week, pidx.day_of_week)
 
         if LooseVersion(pd.__version__) >= LooseVersion("2.0.0"):
```

Comment:
Let's remove this to focus on testing for the latest pandas version.

Reply:
Removed